lib/routes.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/pages/home/home_page.dart';
+import 'package:flutter_survey/pages/login/login_page.dart';
+
+class Routes {
```

Comment:
should you move this to a subdirectory? I think since we are putting a static-class-liked to manage the const, it doesn't need to stay at the same level as `main.dart`

Reply:
Hmm, I prefer to keep it here until we got more complex structure that needs to be optimized. I have no idea where to put this route class right now 🥲